### DISCUSSION TITLE
Contributing to translations of the Saleor

### DISCUSSION THREAD

**Discussion by undercontr** [saleor/saleor#15381] [General] [answered]
Hello, is there any platform like Crowdin to contribute to the translation of the Saleor? I found some problems in the Turkish translation and it's not complete, how can we contribute to the translation?

Another question, is there any possibility to replace the translation provided by the source code? Can we use our own translation for our self-hosted instance?

**Comment by timuric** [comment, marked as answer]
Hi @undercontr you are welcome to contribute to translations https://github.com/saleor/saleor-dashboard it is mostly json file with strings so it should be straightforward https://github.com/saleor/saleor-dashboard/blob/main/locale/tr.json

We will be happy to accept PR!
